apps/dashboard/app/controllers/concerns/batch_connect_concern.rb

```diff
@@ -22,4 +22,16 @@ def bc_dev_app_groups
       apps: nav_dev_apps.select(&:batch_connect_app?)
     )
   end
+
+  def bc_custom_apps_group
+    if !@user_configuration.apps_menu.blank?
```

Comment:
I know you like `blank`, but can we prefer empty for enumerable objects?

```suggestion
    if !@user_configuration.apps_menu.empty?
```

Reply:
It's just the code base has a lot more `empty?`s than `blank?`s, so I'm trying to keep that pattern.

Reply:
Fixed